### PR TITLE
docs(style): [tag] use new display tag

### PR DESCRIPTION
**Pull request by wzc520pyfm**
Related: #10754 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/tag.md [hunk 1]
```diff
@@ -75,23 +75,23 @@ tag/checkable
...
-| round               | whether Tag is rounded               | boolean | —                           | false   |
+| Name                | Description                          | Type                                                     | Default |
+| ------------------- | ------------------------------------ | -------------------------------------------------------- | ------- |
+| type                | component type                       | ^[string]`'success' \| 'info' \| 'warning' \| 'danger'`  | —       |
+| closable            | whether Tag can be removed           | ^[boolean]                                               | false   |
+| disable-transitions | whether to disable animations        | ^[boolean]                                               | false   |
+| hit                 | whether Tag has a highlighted border | ^[boolean]                                               | false   |
+| color               | background color of the Tag          | ^[string]                                                | —       |
+| size                | tag size                             | ^[string]`'large' \| 'default' \| 'small'`               | default |
```
Comment (btea) [inline docs/en-US/component/tag.md:85]: Here type should be `enum` , and the description needs to be synchronized to the props attribute. You can refer to the input.

**Comment by chenxch** [comment]
[Upgrade](https://preview-10754-element-plus.surge.sh/en-US/component/docs)
#10754

If you are interested in participating in this document style upgrade plan, you can view these two documents and claim them.

**Comment by wzc520pyfm** [comment]
@chenxch @btea 
Thank you for your guidance, please checked again. And I'm ready to work on the other component document style.

File: docs/en-US/component/tag.md [hunk 2]
```diff
@@ -73,45 +73,50 @@ tag/checkable
...
-| ----- | ---------------------------- | ---------- |
-| click | triggers when Tag is clicked | —          |
-| close | triggers when Tag is removed | —          |
+### Tag Events
 
-## Slots
+| Name  | Description                  | Type                                                                |
+| ----- | ---------------------------- | ------------------------------------------------------------------- |
+| click | triggers when Tag is clicked | `(evt: MouseEvent)` => ^[Function]`(evt: MouseEvent) => boolean`          |
+| close | triggers when Tag is removed | `(evt: MouseEvent)` => ^[Function]`(evt: MouseEvent) => boolean`          |
```
Comment (tolking) [inline docs/en-US/component/tag.md:96]:
```suggestion
| click | triggers when Tag is clicked | ^[Function]`(evt: MouseEvent) => void`          |
| close | triggers when Tag is removed | ^[Function]`(evt: MouseEvent) => void`          |
```

File: docs/en-US/component/tag.md [hunk 3]
```diff
@@ -73,45 +73,50 @@ tag/checkable
...
 
-## CheckTag Events
+### CheckTag Events
 
-| Name   | Description                        | Parameters |
-| ------ | ---------------------------------- | ---------- |
-| change | triggers when Check Tag is clicked | checked    |
+| Name   | Description                        | Type                                                            |
+| ------ | ---------------------------------- | --------------------------------------------------------------- |
+| change | triggers when Check Tag is clicked | `(value: boolean)` => ^[Function]`(value: boolean) => boolean`    |
```
Comment (tolking) [inline docs/en-US/component/tag.md:117]: | change | triggers when Check Tag is clicked | ^[Function]`(value: boolean) => void` |

File: packages/components/tag/src/tag.vue [hunk 4]
```diff
@@ -44,19 +44,21 @@ const props = defineProps(tagProps)
 const emit = defineEmits(tagEmits)
 
 const tagSize = useFormSize()
-const ns = useNamespace('tag')
-const classes = computed(() => {
+const nsTag = useNamespace('tag')
```
Comment (chenxch) [inline packages/components/tag/src/tag.vue:47]: There is no need to change the name.

File: packages/components/tag/src/tag.vue [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
+    :class="containerKls"
     :style="{ backgroundColor: color }"
     @click="handleClick"
   >
-    <span :class="ns.e('content')">
+    <span :class="contentKls">
       <slot />
     </span>
-    <el-icon v-if="closable" :class="ns.e('close')" @click.stop="handleClose">
+    <el-icon v-if="closable" :class="iconKls" @click.stop="handleClose">
```
Comment (chenxch) [inline packages/components/tag/src/tag.vue:11]: There is only one style that does not need to be transformed with a computed property.

File: packages/components/tag/src/tag.vue (hunk 4 above)
Comment (wzc520pyfm) [reply packages/components/tag/src/tag.vue]: updated

File: packages/components/tag/src/tag.vue (hunk 5 above)
Comment (wzc520pyfm) [reply packages/components/tag/src/tag.vue]: updated

File: packages/components/check-tag/src/check-tag.vue [hunk 6]
```diff
@@ -15,7 +16,11 @@ defineOptions({
 const props = defineProps(checkTagProps)
 const emit = defineEmits(checkTagEmits)
 
-const ns = useNamespace('check-tag')
+const nsCheckTag = useNamespace('check-tag')
```
Comment (chenxch) [inline packages/components/check-tag/src/check-tag.vue:19]: Restore naming.

File: docs/en-US/component/tag.md [hunk 7]
```diff
@@ -73,45 +73,50 @@ tag/checkable
...
+### Tag Attributes
 
-## Events
+| Name                | Description                          | Type                                                  | Default |
+| ------------------- | ------------------------------------ | ----------------------------------------------------- | ------- |
+| type                | type of Tag                          | ^[enum]`'success' \| 'info' \| 'warning' \| 'danger'` | —       |
+| closable            | whether Tag can be removed           | ^[boolean]                                            | false   |
+| disable-transitions | whether to disable animations        | ^[boolean]                                            | false   |
+| hit                 | whether Tag has a highlighted border | ^[boolean]                                            | false   |
+| color               | background color of the Tag          | ^[string]                                             | —       |
```
Comment (chenxch) [inline docs/en-US/component/tag.md:86]: deafult `''`

File: docs/en-US/component/tag.md [hunk 8]
```diff
@@ -73,45 +73,50 @@ tag/checkable
...
-| color               | background color of the Tag          | string  | —                           | —       |
-| size                | tag size                             | string  | large / default /small      | default |
-| effect              | component theme                      | string  | dark / light / plain        | light   |
-| round               | whether Tag is rounded               | boolean | —                           | false   |
+### Tag Attributes
 
-## Events
+| Name                | Description                          | Type                                                  | Default |
+| ------------------- | ------------------------------------ | ----------------------------------------------------- | ------- |
+| type                | type of Tag                          | ^[enum]`'success' \| 'info' \| 'warning' \| 'danger'` | —       |
```
Comment (chenxch) [inline docs/en-US/component/tag.md:82]: add `''`, and default `''`

File: docs/en-US/component/tag.md [hunk 9]
```diff
@@ -73,45 +73,50 @@ tag/checkable
...
+| closable            | whether Tag can be removed           | ^[boolean]                                            | false   |
+| disable-transitions | whether to disable animations        | ^[boolean]                                            | false   |
+| hit                 | whether Tag has a highlighted border | ^[boolean]                                            | false   |
+| color               | background color of the Tag          | ^[string]                                             | —       |
+| size                | size of Tag                          | ^[enum]`'large' \| 'default' \| 'small'`              | default |
```
Comment (chenxch) [inline docs/en-US/component/tag.md:87]: add `''`, and default `''`

**Review by chenxch — APPROVED** [review]
LGTM.🫡

**Review by btea — APPROVED** [review]

**Review by tolking — APPROVED** [review]

**Pull request merged**